egs/commonvoice/asr1/local/filter_text.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import codecs
+from io import open
+import sys
+
+
+PY2 = sys.version_info[0] == 2
```

Comment:
Could you reflect on my comments?
As I mentioned, we'd not support python 2 and you can make your code very simple without these PY2 cases.